main.py

```diff
@@ -12,8 +12,19 @@
 from fastapi.openapi.docs import get_swagger_ui_html, get_swagger_ui_oauth2_redirect_html
 from fastapi.responses import HTMLResponse
 from pydantic import BaseModel
+from typing import Optional
 
 
+class CompilerErrorModel(BaseModel):
```

Comment:
oh, and more of a stylistic choice but I don't usually add `Model` after my models haha (although that conflicts with `ape.exceptions.CompilerError`)

think about what name would be most useful to see for downstream integrators to understand what this model contains